File: SmartDelivery/app/src/main/java/com/example/smartdelivery/ui/main/view/activity/MenuActivity.java
package com.example.smartdelivery.ui.main.view.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.smartdelivery.R;
import com.example.smartdelivery.ui.main.single.ArriveActivity;

import androidx.appcompat.app.AppCompatActivity;

public class MenuActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_menu);
        Button btn_arrive=findViewById(R.id.menuarrive);
        btn_arrive.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent=new Intent(MenuActivity.this, ArriveActivity.class);
                startActivity(intent);
            }
        });
        findViewById(R.id.menuinvoice).setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MenuActivity.this, MainActivity.class);
                startActivity(intent);
            }
        });

    }


}

File: SmartDelivery/app/src/main/java/com/example/smartdelivery/ui/main/view/activity/MenuActivity.kt
package com.example.smartdelivery.ui.main.view.activity

import android.content.Intent
import com.example.smartdelivery.R
import com.example.smartdelivery.databinding.ActivityMenuBinding
import com.example.smartdelivery.ui.base.BaseActivity

class MenuActivity : BaseActivity<ActivityMenuBinding>(R.layout.activity_menu) {

    override fun init() {
        super.init()

        binding.menuarrive.setOnClickListener {
            val intent= Intent(this, PackagesActivity::class.java)
            startActivity(intent);
        }
        binding.menuinvoice.setOnClickListener {
            val intent= Intent(this, MainActivity::class.java)
            startActivity(intent);
        }
        binding.menuloss.setOnClickListener{
            val intent=Intent(this,LossActivity::class.java)
            startActivity(intent)
        }
        binding.btnOption.setOnClickListener {
            startActivity(Intent(this, OptionActivity::class.java))
        }


    }


}